docker-compose.yml

```
@@ -120,18 +120,47 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  # Different FerretDB v1 instances must use different POSTGRES_DB to avoid error such as
+  # `relation "_ferretdb_database_metadata" already exists (SQLSTATE 42P07)`.
   postgres:
-    image: ${POSTGRES_IMAGE:-postgres}
+    image: postgres
     restart: on-failure
-    ports:
-      - 5432:5432
     environment:
       # UTC−03:30/−02:30. Set to catch timezone problems.
       - TZ=America/St_Johns
-      - POSTGRES_USER=postgres_user
-      - POSTGRES_PASSWORD=postgres_password
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
       - POSTGRES_DB=dance
 
+  postgres-documentdb:
+    image: ${POSTGRES_IMAGE:-ghcr.io/ferretdb/postgres-documentdb:17-0.102.0-ferretdb-2.0.0}
+    platform: linux/amd64
+    restart: on-failure
+    environment:
+      # UTC−03:30/−02:30. Set to catch timezone problems.
+      - TZ=America/St_Johns
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=postgres
+
+  ferretdb2:
+    image: ${FERRETDB_IMAGE:-ghcr.io/ferretdb/ferretdb:2.0.0}
```

```suggestion
    image: ${FERRETDB_IMAGE:-ghcr.io/ferretdb/ferretdb-dev:main-prod}
```

From https://github.com/FerretDB/FerretDB/pull/4911

---

Thank you :pray: